sts_playground/autoencoder/tf_lib.py

```diff
@@ -300,12 +302,17 @@ def train(
     gc.collect()  # clean up old unnecessary data
 
     # train_ds = train_ds.shuffle(train_size, reshuffle_each_iteration=True)
-    train_ds = train_ds.batch(batch_size, drop_remainder=True)
-    valid_ds = valid_ds.batch(batch_size, drop_remainder=True)
+    train_ds = train_ds.batch(batch_size, drop_remainder=False)
```

Comment:
Why set drop_remainder to False?

Reply:
Mistake, this was a hold-over from when I could only load 10k datapoints at a time, and I wanted to make the most of it.